lib/media/streaming_engine.js

```diff
@@ -1262,6 +1262,17 @@ shaka.media.StreamingEngine = class {
             mediaState.stream.segmentIndex.getIteratorForTime(presentationTime);
         ref = mediaState.segmentIterator &&
             mediaState.segmentIterator.next().value;
+        // For LL-HLS we need to find the nearest INDEPENDENT segment
```

Comment:
Oh hey, while examining this code, I found a JSDoc problem!

In the definition for `shaka.media.StreamingEngine.MediaState_`, we refer to `segmentIterator` as being a `shaka.media.SegmentIndexIterator` object at one point. That should be `shaka.media.SegmentIterator`

Reply:
I think that the definition is true, because it's used in other parts and it works, It's a private definition

Reply:
I'm not sure... but it works :S